rpc/json-rpc/README.md

```diff
@@ -73,4 +73,65 @@ assert_eq!(json, expected_json);
 
 This is how the method/param types are done in Cuprate.
 
-For reasoning, see: <https://github.com/Cuprate/cuprate/pull/146#issuecomment-2145734838>.
\ No newline at end of file
+For reasoning, see: <https://github.com/Cuprate/cuprate/pull/146#issuecomment-2145734838>.
+
+### Serialization changes
+This crate's serialized field order slightly differs compared to `monerod`.
+
+`monerod`'s JSON objects are serialized in alphabetically order, where as this crate serializes the fields in their defined order (due to [`serde`]).
+
+With that said, parsing should be not affected at all since a key-value map is used:
```

Comment:
Don't think this matters, but I realized using `serde` means our fields are serialized in definition order, not alphabetical like `monerod`.

Reply:
good spot, but should be fine